src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1015,6 +1015,13 @@ namespace winrt::TerminalApp::implementation
                     if (profileGuid.has_value())
                     {
                         const auto settings{ winrt::make<TerminalSettings>(_settings, profileGuid.value(), *_bindings) };
+                        const auto workingDirectory = terminalTab->GetActiveTerminalControl().WorkingDirectory();
+                        const auto validWorkingDirectory = !workingDirectory.empty();
```

Comment:
I relaxed the validation per the comments by @zadjii-msft in https://github.com/microsoft/terminal/issues/8166#issuecomment-724678264

I'm not sure how we handle `wt -d c:\some\invalid\path` nowadays. Could use some help here.

Reply:
So currently, `wt -d c:\some\invalid\path` will do this:

![image](https://user-images.githubusercontent.com/18356694/102246700-a8b6da80-3ec4-11eb-8979-16a5c8584422.png)


That's not a super useful error, `ERROR_DIRECTORY` - I guess that makes sense, though not totally obvious what's wrong here. 

I'm okay with us leaving this for now. We do some validation of the `startingDirectory` on settings load, but nothing for directories passed as commandline args, and I think that's fine.

If we did want to try and pre-validate, we could pull `Profile::EvaluateStartingDirectory` into a helper, and only use the control's directory if it's valid. That's actually probably a good idea.